### PR TITLE
DxpClass for Dxp detectors

### PR DESCRIPTION
**Pull request by gabrielfedel**
Hello Hugo, I added a DxpClass to permit to use Dxp detectors.

### CODE REVIEW

File: py4syn/epics/DxpClass.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+            self.saveSpectrum(c, **kwargs)
+            return 1.0
+
+    # save the spectrum intensity in a mca file
+    def saveSpectrum(self, ch, **kwargs):
+        fileName = self.fileName
+        idx = 0
+        if(fileName):
+            spectrum = self.pvDxpChannels[ch].get(as_numpy=True)
+            if fileName[-4] == '.':
```
Comment (hhslepicka) [inline py4syn/epics/DxpClass.py]: Just a small comment here:
You don't need to check for the presence of the `.` at the file name, you can just do the split directly and move forward with that.  This way you can avoid the repeated code:

```
    prefix = fileName.split('.')[0]
    while os.path.exists('%s_%s%d_%04d.mca' % (prefix, self.dxpType, ch, idx)):
        idx += 1
    fileName = '%s_%s%d_%04d.mca' % \
                                  (prefix, self.dxpType, ch, idx)
```

**Comment by hhslepicka** [comment]
Hi @gabrielfedel , thanks for your contribution.
It looks pretty good to me! 👍 
I just added a small comment between lines 119 and 131.
Let me know what you think about it.

**Comment by gabrielfedel** [comment]
Hi Hugo! 

I couldn't find this comment...I need to give you access to my repository?

**Review by hhslepicka — CHANGES_REQUESTED** [review]

**Comment by hhslepicka** [comment]
@gabrielfedel Sorry, I forgot to submit the request for changes.

**Comment by gabrielfedel** [comment]
Great suggestion, thank you!
I did it!

**Review by hhslepicka — APPROVED** [review]

**Comment by hhslepicka** [comment]
Looks good to me! Thank you @gabrielfedel .

**Pull request merged**